proposals/IDE/NSBP2020.md

```diff
@@ -0,0 +1,29 @@
+# Title
+Astropy Ambassadors at NSBP 2020
+
+## Summary
+The Astropy Project runs workshops and outreach efforts at meetings such as the American Astronomical Society meetings. It would be beneficial to the Project (in terms of growing a more diverse contributor and user base) to expand our reach to include meetings of other professional societies, especially prioritizing meetings that serve underrepresented minorities. One relevant professional society and conference is the National Society for Black Physicists (NSBP). As a first step toward a more concrete partnership with the NSBP, we propose to attend the NSBP 2020 as exhibitors.
+
+## Team
+- [Adrian Price-Whelan](https://github.com/adrn)
+- Anyone else who is interested to attend NSBP 2020! (**Please "suggest changes" to the Pull Request or leave a comment to be added**)
```

Comment:
I am interested in attending NSBP `[Robel Geda](https://github.com/robelgeda)` - Thank you, Robel.